pages/activities.jsx

```diff
@@ -162,6 +162,14 @@ var ActivitiesPage = React.createClass({
               link="/activities/privacy-basics/"
               developedBy="Stacy Martin - Senior Data Privacy Manager at Mozilla and the Mozilla Learning Network team"
               description="Learn how to safeguard your privacy online and develop an awareness of how companies and governments track and collect your data online."/>
+	        <ActivityKit
```

Comment:
this will need to be indented back a little, to match the code above and below it